MapboxCoreNavigation/NavigationRouteOptions.swift

```diff
@@ -25,7 +25,11 @@ open class NavigationRouteOptions: RouteOptions {
         shapeFormat = .polyline6
         includesSteps = true
         routeShapeResolution = .full
-        attributeOptions = [.congestionLevel, .expectedTravelTime, .maximumSpeedLimit]
+        if profileIdentifier == .walking {
+            attributeOptions = [.congestionLevel, .expectedTravelTime]
+        } else {
+            attributeOptions = [.congestionLevel, .expectedTravelTime, .maximumSpeedLimit]
```

Comment:
Nit: A less repetitive way of writing this conditional code would be:

```swift
attributeOptions = [.congestionLevel, .expectedTravelTime]
if profileIdentifier == .automobile || profileIdentifier == .automobileAvoidingTraffic {
    attributeOptions.insert(.maximumSpeedLimit)
}
```

Let’s add a unit test to ensure that the right `attributeOptions` results for the profiles we care most about.